src/util/init_ini_data.cpp

```diff
@@ -451,8 +452,14 @@ namespace hpx { namespace util
             return plugin_registries;
 
         // make sure each node loads libraries in a different order
+#if defined(HPX_HAVE_CXX11_STD_SHUFFLE)
+        std::random_device random_device;
+        std::mt19937 generator(random_device());
+        std::shuffle(libdata.begin(), libdata.end(), std::move(generator));
+#else
```

Comment:
Well, since your test for  `HPX_HAVE_CXX11_STD_SHUFFLE`  includes  `std::random_device`  we should be fine as it is.